Marble Blast Platinum/platinum/client/scripts/replay.cs

```diff
@@ -20,7 +20,7 @@
 // DEALINGS IN THE SOFTWARE.
 //-----------------------------------------------------------------------------
 
-$ReplayVersion = 25;
+$ReplayVersion = 26;
```

Comment:
Technically, this isn't needed and it should still replay just as it did in older PQ versions.